<launch>
    <!-- Autogenerated launch file - do not hand edit -->
    <include file="{{machine_file}}" />
    {% for namespace, topic, machine in blob_finder_launch %}
    <group ns="{{namespace}}">
        <node pkg="mct_blob_finder" type="blob_finder_node.py" name="blob_finder" args="{{topic}} {{skip_param}}" machine="{{machine}}" />
        {% if display %}
        <node pkg='image_view' type="image_view" name="image_view">
            <remap from="image" to="{{namespace}}/image_blobs"/>
        </node>
        {% endif %}
    </group>
    {% endfor %} 
</launch>
